src/actions/multiple_target_action.py

```diff
@@ -46,3 +39,15 @@ def cursorless_multiple_targets(m) -> str:
             target_list = target_list + [STRICT_HERE]
 
     return target_list
+
+
+def update_target(target: dict, value: dict):
+    type = target["type"]
+    if type == "list":
+        for target in target["elements"]:
+            update_target(target, value)
+    # elif type == "range":
+        # update_target(target["start"], value)
+        # update_target(target["end"], value)
```

Comment:
for the generic function, I think this is probably the right behaviour.  Shall we keep it?

Reply:
actually maybe we should just operate on first thing and then let inference do the rest of the work

Reply:
so if list, operate on first list element, if range operate on start

Reply:
and then i think extension-side inference will do its work from there

Reply:
k i'll take this one over